go/multiorch/recovery/actions/fix_replication.go

```diff
@@ -436,6 +445,98 @@ func (a *FixReplicationAction) Priority() types.Priority {
 	return types.PriorityHigh
 }
 
+// fixDivergedTimeline repairs a replica that has a diverged timeline using pg_rewind.
+//
+// The flow is:
+// 1. Run pg_rewind --dry-run via multipooler to check if repair is feasible
```

Comment:
why doing in this check? In our context it seems we want to do it and if the actual run fails, the outcome is the same for us.